deepcell_toolbox/metrics.py

```diff
@@ -598,6 +604,11 @@ def save_to_dataframe(self):
         Returns:
             pandas.DataFrame: Single row dataframe with error types as columns
         """
+
+        self.precision = self.correct_detections / self.n_pred
+        self.recall = self.correct_detections / self.n_true
+        self.f1 = hmean([self.recall, self.precision])
```

Comment:
Do we really need `hmean`?

Could we just do:

```python
self.f1 = 2*(self.recall * self.precision) / (self.recall + self.precision)
```

Reply:
Yeah, the reason I was using hmean was because Dave used to compute F1 back when we had to do it on the output for the Metrics package. Now that we're doing it internally on each loop, we only have two values, rather than a list of like 300 values each. I'll swap it.

Reply:
Reflecting on it, I see no need to swap it out. I'm happy with the `hmean` build.